db/migrate/20110317144932_add_session_table.rb

```diff
@@ -1,7 +1,7 @@
 class AddSessionTable < ActiveRecord::Migration[4.2]
 
 	class Event < ApplicationRecord
-		serialize :info
+		serialize :info, coder: YAML
```

Comment:
Can we temporarily follow the existing pattern - even if it's not super required here, more so for future travellers that might be doing rails upgrades here